src/SFA.DAS.FindEmploymentSchemes.Web/Views/Shared/_Layout.cshtml

```diff
@@ -36,7 +36,10 @@
       <script asp-add-nonce>
           document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');
       </script>
-      <div id="layout-cookie-banner" class="govuk-cookie-banner " data-nosnippet role="region" aria-label="Cookies on Find schemes for your business">
+
+      <form id="layout-cookie-banner-form" action="#" method="post" novalidate="">
```

Comment:
Is this form needed?

Reply:
No you're right, I forgot to remove that from my first attempt. Will remove.